CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This changelog is a work in progress and may contain notes for versions which have not actually been released. Check the [Releases](https://github.com/0xProject/0x-mesh/releases) page to see full release notes and more information about the latest released versions.
 
+## v7.1.0-beta
+
+### Features ✅
+
+ - Reduce startup time for Mesh node by only waiting for a block to be processed if Mesh isn't already sync'ed up to the latest block. ([#622](https://github.com/0xProject/0x-mesh/pull/622))
+ - Developers can now override the contract addresses for any testnet using the `CUSTOM_CONTRACT_ADDRESSES` env config ([#640](https://github.com/0xProject/0x-mesh/pull/640)).
+  - Add `getOrdersForPageAsync` method to `@0x/mesh-rpc-client` WS client interface so that clients can paginate through the retrieved orders themselves ([#642](https://github.com/0xProject/0x-mesh/pull/642)).
+
+### Bug fixes 🐞
+
+ - Fixed a bug where we attempted to update the same order multiple times in a single DB txn, causing the later update to noop. ([#623](https://github.com/0xProject/0x-mesh/pull/623)).
```

Comment:
Should we backport the `DevUtils` fixes? It looks like we updated the address of mainnet DevUtils, so we should probably add a bullet to the `Bug fixes` section.

Reply:
The updates to `DevUtils` only work with protocol v3 right? If we wanted to do this we would need to also deploy a new version of `DevUtils` for protocol v2.

Reply:
We still updated `DevUtils`, so I think we should add a bullet to the changelog.

Reply:
Hmm.. I think that might have been a mistake. Made a separate comment about it.

Reply:
It was a mistake. Fixed it.